services/QuillLMS/engines/evidence/app/models/evidence/activity_health.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+module Evidence
+  class ActivityHealth < ApplicationRecord
+    ALLOWED_TOOLS = %w(connect grammar)
+    FLAGS = [
```

Comment:
Please source this data from:
https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/models/concerns/flags.rb

Otherwise, when `flags.rb` is updated with new entries, this class will fail validation.